### ISSUE TITLE
Archive or update?

### ISSUE THREAD

**Issue by jogomojo** [Xilinx/PYNQ_RFSOC_Workshop#11]
As discussed in [this thread](https://discuss.pynq.io/t/pynq-rfsoc-workshop-image-3-0-1-for-zcu111-does-not-contain-the-notebooks/7521) on the PYNQ forum, this repo no longer installs properly on the PYNQ 3.0 image. 

We should consider either archiving this repo or upgrading it to work with the newest version of PYNQ. 

My suggestion is to archive it as more comprehensive material is available in the [RFSoC book repo](https://github.com/strath-sdr/RFSoC-Book).

**Comment by schelleg** [comment]
agreed - please archive it with a nice message and pointer to the RFSoC book ...

README would get updated with something similar to archived: https://github.com/Xilinx/ResNet50-PYNQ
